src/scanner.cc

```diff
@@ -1725,11 +1730,14 @@ Result newline_indent(uint32_t indent, State &state) {
  */
 Result newline_token(uint32_t indent, State &state) {
   // TODO(414owen): fix
-  return
-    (peeks(cond::symbolic | cond::ticked)(with(read_symop)(newline_infix(indent)) + fail) +
-    newline_where(indent) +
-    peek('i')(in)
-    )(state);
+  // TODO Convert to switch
+  if (cond::symbolic(PEEK) || PEEK == '`') {
```

Comment:
2 peeks happen here, store in a variable? Or use a `switch(PEEK)` like the TODO mentions :)